pkg/controllers/provisioning/scheduling/metrics.go

```diff
@@ -78,4 +78,14 @@ var (
 			ControllerLabel,
 		},
 	)
+	PodSchedulerDecisionSeconds = opmetrics.NewPrometheusHistogram(
```

Comment:
I think we need a gauge metric that is paired with this to track the amount of time that we have a pod but the pod doesn't have a decision made against it

Reply:
Also, looks like you also declared this metric in cluster state -- did you intend to declare the metric there as well?

Reply:
> I think we need a gauge metric that is paired with this to track the amount of time that we have a pod but the pod doesn't have a decision made against it 

This would have to be a separate thread, yeah?